scripts/configure-macos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+die() {
+    echo "$(basename $0): $@" >&2
+    exit 1
+}
+
+test -d flux-env || die "please run from the top of the source tree"
```

Comment:
Since there's a `flux-env` command, maybe naming this directory `macos-venv` or something would be more clear? Also, the user may have just not yet run `install-deps-macos.sh`, so maybe the error message should suggest that, e.g. "Ensure you've run scripts/install-deps.macos.sh and are in the top-level source directory"